pages/index.js

```diff
@@ -1,7 +1,12 @@
+import Footer from "@/components/Footer/Footer";
+import Header from "@/components/Header/Header";
+import styled from "styled-components";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <Header />
+      <Footer />
```

Comment:
since you have a Layout component by now, you can remove those two lines of code and replace it with content.
you can also remove the imports in lines 1 and 2.

(there needs to be something your component returns, like an h1 element)